elasticsearch/docker-entrypoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+export PATH=$PATH:/usr/share/elasticsearch/bin/
+
+# Add elasticsearch as command if needed
+if [ "${1:0:1}" = '-' ]; then
+	set -- elasticsearch "$@"
+fi
+
+# Drop root privileges if we are running elasticsearch
+# allow the container to be started with `--user`
+if [ "$1" = 'elasticsearch' -a "$(id -u)" = '0' ]; then
+	# Change the ownership of user-mutable directories to elasticsearch
+	for path in \
+		/usr/share/elasticsearch/data \
+		/usr/share/elasticsearch/logs \
```

Comment:
This seems like our elasticsearch implementation is broken as these are stateless directories and so I'd expect the data and logs to be configured under /var.

Reply:
Yes, you are right, I will change them to /var